source/frontend/src/components/maps/MapLeafletView.tsx

```diff
@@ -159,12 +157,12 @@ const MapLeafletView: React.FC<React.PropsWithChildren<MapLeafletViewProps>> = (
           type: 'Feature',
           properties: {},
         };
-        if (mapLocationFeatureDataset.parcelFeature !== null) {
-          activeFeature = mapLocationFeatureDataset.parcelFeature;
+        if (mapLocationFeatureDataset.parcelFeatures !== null) {
+          activeFeature = mapLocationFeatureDataset.parcelFeatures[0];
```

Comment:
thinking through this - should we be showing all of the parcels that were at the click location?

Reply:
Its a possible solution. I think the story about map interactions will guide and solve that issue.